Dockerfile

```diff
@@ -1,12 +1,9 @@
 FROM sameersbn/ubuntu:14.04.20170503
 MAINTAINER sameer@damagehead.com
 
-ENV GITLAB_VERSION=9.1.4 \
+ENV GITLAB_VERSION=9.2.0 \
     RUBY_VERSION=2.3 \
     GOLANG_VERSION=1.6.3 \
-    GITLAB_SHELL_VERSION=5.0.2 \
```

Comment:
Yeah, the only reason I added this is that it's a pain to Google the separate latest gitlab versions, and which ought to be used for which build.

This way, rebuilding the container for any GitLab version just involves changing a single variable and it'll pull the artifacts compatible with that version, per the included _VERSION files.

It's not a big deal for me -- I can add them back in and update to the latest; just thought it would ease your maintenance in the future.